fdbserver/tester.actor.cpp

```diff
@@ -1651,7 +1651,7 @@ ACTOR Future<Void> runTests(Reference<AsyncVar<Optional<struct ClusterController
 
 		if (perpetualWiggleEnabled) { // restore the enabled perpetual storage wiggle setting
 			printf("Set perpetual_storage_wiggle=1 ...\n");
-			wait(setPerpetualStorageWiggle(cx, true, LockAware::True));
```

Comment:
Would you mind telling what's the reason we need a commit version here?

Reply:
With a unfortunate timing, the following sequences were observed:
1. The test workload completes.
2. QuietDatabase starts, and `setPerpetualStorageWiggle(false)` is called to disable perpetual wiggle, which generates a transaction.
3. One storage server pulls the `disable perpetual wiggle` transaction, and applies it to the PTree
4. QuietDatabase tries to obtain the storage queue size, and it obtains the `StorageMetrics` trace event the above storage server, before the `disable perpetual wiggle` mutation is committed to the storage engine.
5. QuietDatabase fails since the storage queue is not zero.
6. repeat from Step 2.

With the commit version from `setPerpetualStorageWiggle()`, we can extract the storage queue size only after the `disable perpetual wiggle` mutation is durable. This is not the only solution, but it seems to be quite stable. I am open to other fixes.